src/bindings/generator/test/test_dbus_typing.py

```diff
@@ -0,0 +1,22 @@
+# todo: write proper tests using those cases
+# if __name__ == "__main__":
+#    # success cases
+#    print(map_input_param_to_pytype("ss"))
+#    print(map_input_param_to_pytype("ss(xi)"))
+#    print(map_input_param_to_pytype("ssa{xi}"))
```

Comment:
usually is not good to merge dead code but I am assuming a new patch is coming soon.